common/src/main/java/org/terracotta/angela/common/AngelaProperties.java

```diff
@@ -28,9 +28,9 @@
 @SuppressFBWarnings("DMI_HARDCODED_ABSOLUTE_FILENAME")
 public enum AngelaProperties {
   // root dir where Angela puts installation, work directories and any file that is needed
-  ROOT_DIR("angela.rootDir", Paths.get("/data/angela").toAbsolutePath().toString()),
+  ROOT_DIR("angela.rootDir", Paths.get(System.getProperty("user.home"), "angela").toAbsolutePath().toString()),
```

Comment:
can you remove this change and the one below for now ? this could break CI jobs - if we want to change this we'll need to review our CI first to make sure we override the property

Reply:
Other option is to now make this sysprop required for 3.2.x ?

This is a prop that is not set in jenkins but in the Gradle or Maven build system per test module.

Since we go through all the projects anyway. For tc-platform, angela-ee and tc-ee, all are setting this prop.